tsconfig.server.json

```
@@ -0,0 +1,17 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "lib": ["ES2019"],
+    "target": "ES2019",
```

Would esnext work here?

```suggestion
    "lib": ["esnext"],
    "target": "esnext",
```